### ISSUE TITLE
AVT 1 - Combobox has DAP violations for disabled and invalid knobs

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4530]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="1016" alt="Screen Shot 2019-11-01 at 2 27 37 PM" src="https://user-images.githubusercontent.com/21676914/68050865-10579980-fcb4-11e9-9744-c05c3feec75b.png">
Note: The title and optional text of the Combobox have a style change, but nothing to indicate to screen reader users the disabled state, such as aria-disabled="true".

Deselect the disable knob and select invalid and the following violations are present:
<img width="1002" alt="Screen Shot 2019-11-01 at 2 35 55 PM" src="https://user-images.githubusercontent.com/21676914/68051323-26b22500-fcb5-11e9-9cc4-fc83ccf6d771.png">

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
